### PR TITLE
Requester profile 

### PR DESCRIPTION
**Pull request by Macbull**
Adding some fields to serializer for requester profile

### CODE REVIEW

File: crowdsourcing/models.py [hunk 1]
```diff
@@ -140,9 +140,9 @@ class Project(models.Model):
     name = models.CharField(max_length=128, error_messages={'required': "Please enter the project name!"})
     start_date = models.DateTimeField(auto_now_add=True, auto_now=False)
     end_date = models.DateTimeField(auto_now_add=True, auto_now=False)
-    owner = models.ForeignKey(Requester, related_name='project_owner')
+    owner = models.ForeignKey(Requester, related_name='projects_owned')
```
Comment (dmorina) [inline crowdsourcing/models.py]: wrong related name

File: crowdsourcing/models.py [hunk 2]
```diff
@@ -140,9 +140,9 @@ class Project(models.Model):
...
     description = models.CharField(max_length=1024, default='')
-    collaborators = models.ManyToManyField(Requester, through='ProjectRequester')
+    collaborators = models.ManyToManyField(Requester, through='ProjectRequester', related_name='projects_collaborated')
```
Comment (dmorina) [inline crowdsourcing/models.py]: collaborators :)

File: crowdsourcing/serializers/requester.py [hunk 3]
```diff
@@ -2,10 +2,26 @@
 
 from crowdsourcing import models
 from rest_framework import serializers
+import datetime
 
 class RequesterSerializer(serializers.ModelSerializer):
+    total_projects = serializers.SerializerMethodField()
+    current_projects = serializers.SerializerMethodField()
+    waiting_projects = serializers.SerializerMethodField()
+
```
Comment (dmorina) [inline crowdsourcing/serializers/requester.py]: these functions are supposed to live under module

File: crowdsourcing/models.py (hunk 1 above)
Comment (Macbull) [inline crowdsourcing/models.py]: I just thought tom make them more intuitive to use while using back-relations

File: crowdsourcing/serializers/requester.py (hunk 3 above)
Comment (Macbull) [inline crowdsourcing/serializers/requester.py]: but then we will need to pass extra parameter to module serializer ! 
PS : inspired from Worker REST API

File: crowdsourcing/models.py (hunk 1 above)
Comment (dmorina) [inline crowdsourcing/models.py]: usually it's the related models combined

File: crowdsourcing/serializers/requester.py (hunk 3 above)
Comment (dmorina) [inline crowdsourcing/serializers/requester.py]: yeah, eventually worker would have to be updated to, let's not break the concept, it's ok to have a method in the module viewset which gets by requester or by worker

File: crowdsourcing/models.py (hunk 1 above)
Comment (Macbull) [inline crowdsourcing/models.py]: oh! ok ! I will update these soon !

File: crowdsourcing/serializers/requester.py (hunk 3 above)
Comment (dmorina) [inline crowdsourcing/serializers/requester.py]: A serializer mainly deals with it's own Model and additional ones if any

Comment (Macbull) [inline crowdsourcing/serializers/requester.py]: but these are only counts specific to a requester,  not specific to any module or project

Comment (dmorina) [inline crowdsourcing/serializers/requester.py]: it doesnt matter, what matters is the object type and these are a list of modules so they go under ModuleSerializer

**Pull request closed without merging**